aiohttp/web_request.py

```diff
@@ -457,9 +458,22 @@ def content(self):
 
     @property
     def has_body(self):
-        """Return True if request has HTTP BODY, False otherwise."""
+        """Return True if request's HTTP BODY can be read, False otherwise."""
+        warnings.warn(
+            "Deprecated, use .can_read_body #2005",
+            DeprecationWarning, stacklevel=2)
+        return not self._payload.at_eof()
```

Comment:
Added old attr back into test cases - is this okay or should I break it out into another test case?